2.6.md

```diff
@@ -2294,6 +2299,16 @@ This object provides information pertaining to the device through which the user
     <td><code>macmd5</code></td>
     <td>string; DEPRECATED</td>
     <td>MAC address of the device; hashed via MD5.</td>
+ <tr>
+   </tr>
+    <td><code>brand</code></td>
+    <td>string</td>
+    <td>Brand of the device; Many mobile device manufacturers subdivide their devices into brands (eg. Samsung Galaxy S-Series, Samsung Galaxy A-Series), each of which may have different capabilities and limitations.</td>
+ <tr>
+   </tr>
+    <td><code>os</code></td>
```

Comment:
Isn't Device.os an existing OpenRTB field?

Reply:
Apologies - yes it is